backend/api/users.py

```diff
@@ -147,29 +156,18 @@ def insert_bookmarks(id):
 @users.route('/api/users/bookmarks/<uuid:id>', methods=['DELETE'])
 @login_required
 def delete_bookmarks(id):
-    user = current_user
-    if user is None:
-        return make_response("No current user detected ", 400)
-    res = g.user_datastore.get_user(user['email'])
-    if not res:
-        return make_response("Unknown User with Email-address: " +
-                             user['email'], 400)
-
+    res = g.user_datastore.get_user(current_user['email'])
     if id in res.bookmarks:
         res.bookmarks.remove(id)
         res.save()
         return jsonify(res['bookmarks'])
-    return make_response("Project is not bookmarked: ", 400)
+    return make_response("Project is not bookmarked: " + id, 400)
```

Comment:
it raised a 500 error because flask didn't handle the exception (because it can't implitly cast). 

changed it to str(id) now and I don't see a reason why it wouldn't work now, it's only the response text anyways.